src/index.html

```diff
@@ -2,16 +2,148 @@
+          method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label>
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label>
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label>
+          Full date of birth:
+          <input
+            type="date"
+            name="date-of-birth"
+            required
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label>
+          E-mail:
+          <input
+            type="email"
+            name="mail"
+            required
+            minlength="5"
+            maxlength="40"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password: <input type="password"
+                          name="password"
+                          required
+                    >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you! </legend>
+        Do you love cats?
+        <label class="diff">
+          <input type="radio"
+                 name="catlover"
+                 value="yes"
+          >
+          Yes
+        </label>
+        <label class="diff">
+          <input type="radio"
+                 name="catlover"
+                 value="no"
+          >
+          No
+        </label>
+        <label>
+          What is your favorite color?
+          <input type="color"
+                 name="color"
+          >
+        </label>
+        <label>
+          What time do you go to bed?
+          <input type="time"
+                 name="time-to-bed"
+          >
+        </label>
+        <label>
+          What are your favorite brand of cars?
+          <select name="brand" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label>
+          How do you rate our work?
+          <input type="range"
+                 name="work-rate"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info: </legend>
+        <label>
+          Coments:
+          <textarea name="coments">
+          </textarea>
```

Comment:
To fix empty spaces in textarea:
<img width="294" alt="image" src="https://user-images.githubusercontent.com/118751034/214657686-a78ab4e0-1194-498a-8c59-c73d6a0e85ff.png">

```suggestion
          <textarea name="coments"></textarea>
```